config/v2_3_experimental/types/disk.go

```diff
@@ -156,7 +156,7 @@ func (n Disk) partitionsUnitsMismatch() bool {
 		if p.Size != nil || p.Start != nil {
 			partsNotInMb = true
 		}
-		if p.SizeMb != nil || p.StartMb != nil {
+		if p.SizeMiB != nil || p.StartMiB != nil {
```

Comment:
Would it make sense to change the other variable names in this function (`partsInMb` & `partsNotInMb`) for clarity as well?

Reply:
Sure, but I it's not critical since it's not user facing. Would rather do that in a separate PR so I can unblock the spec bump and get that out the door sooner.

Reply:
PR's good to me either way